src/Adapter/MSTest.TestAdapter/Execution/UnitTestRunner.cs

```diff
@@ -363,9 +413,9 @@ private class ClassCleanupManager
             _reflectHelper = reflectHelper ?? new ReflectHelper();
         }
 
-        public void MarkTestComplete(TestMethodInfo testMethodInfo, TestMethod testMethod, out bool shouldCleanup)
+        public void MarkTestComplete(TestMethodInfo testMethodInfo, TestMethod testMethod, out bool shouldClassCleanup, out bool shouldAssimplyCleanup)
```

Comment:
Rename `shouldClassCleanup` with `shouldRunClassCleanup` and `shouldAssimplyCleanup` with `shouldRunAssemblyCleanup`.